src/reader/slice_reader.rs

```diff
@@ -16,24 +14,10 @@ use crate::reader::{is_whitespace, BangType, ReadElementState, Reader, Span, Xml
 
 use memchr;
 
-/// This is an implementation for reading from a `&[u8]` as underlying byte stream.
-/// This implementation supports not using an intermediate buffer as the byte slice
-/// itself can be used to borrow from.
+/// This is an implementation of [`Reader`] for reading from a `&[u8]` as
```

Comment:
Actually, this is a docstring for a `Reader` struct and this reference would be to the same page. Not much useful as for me.